src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx

```diff
@@ -1,28 +1,46 @@
 import * as _ from 'lodash';
 import * as React from "react";
 
+import WebMercatorViewport from 'viewport-mercator-project';
+import bbox from '@turf/bbox';
+import * as turf from '@turf/turf'
```

Comment:
Turf.js as a whole is a big library, I recommend importing these two specific modules to make your page faster -

`import helpers, nearest-point-on-line as turf from '@turf/turf'`

Reply:
Added some comments around the func rankRiders.